chaoslib/litmus/container_kill/containerd_chaos/crictl-chaos.yml

```diff
@@ -17,7 +45,7 @@
         executable: /bin/bash
       register: crictl_ds_result
 
-    - name: Confirm that the containerd-chaos ds is running on all nodes.
+    - name: Confirm that the containerd-chaos job is running on {{ app_node }} node.
```

Comment:
Previously, We are putting  `ignore_errors: true` for this task because we are unable to check the running status of the DS. But now it is changed to Job so it is easy to check the status so can you please update this task and remove ignore_error for this task?